client/build/components/AssetPreview/index.js

```diff
@@ -19,7 +19,7 @@ var AssetPreview = function AssetPreview(_ref) {
       fileExt = _ref$claimData.fileExt,
       contentType = _ref$claimData.contentType,
       thumbnail = _ref$claimData.thumbnail;
-  var directSourceLink = "".concat(claimId, "/").concat(name, ".").concat(fileExt);
+  var directSourceLink = "asset/".concat(name, "/").concat(claimId);
```

Comment:
not a stopper, but it would be ideal to stick with single or double quotes, I see some mixed formatting

Reply:
@billbitt I saw #405 was closed but if you add Prettier I believe this will never be an issue again.

Reply:
We should add Prettier (I reopened #405), but these files are the build result from babel transpiling the es6 code.
I'd like to avoid checking the built code into version control, but we have to include it at this point so that it can be imported by other packages like `lbryio/www.spee.ch`.  I'd like merge `lbryio/www.spee.ch` and `lbryio/spee.ch` (#516) which should result in being able to have one build process that does not need to check built code into version control.